terminology.md

```diff
@@ -30,10 +25,6 @@ END_METADATA -->
   This is one of the API keys that is retrieved by logging in on
   [portal.vipps.no](https://portal.vipps.no).
 
-* D-number - A Norwegian identification number given to foreign persons who
-  do not have a Norwegian national identity number (NIN), but have a need for
-  identification for the Norwegian authorities.
-
 * Deeplink - The Vipps "deeplink" opens the payment page in Vipps where the user
```

Comment:
Should Vipps be changed to Vipps MobilePay instead? It is used several times on the page

Reply:
@HenrietteBossen Thank you! I updated it to remove the name "Vipps" all together, and I changed "opens the payment page in Vipps" to "opens the payment page in the app". That way, it can be applicable to the MobilePay app, too... Does that seem to make sense?

Reply:
I will merge it now, but we can update it more afterwards. :-)